README.md

```diff
@@ -1,206 +1,68 @@
-# Notary Project Overview
+# Notary Project Specifications
 
-A current overview of Notary v2, including a collection of requirements & scenarios, framing the scope of the Notary v2 project.
+This repository is in active maintenance and contains specifications shared across repositories under Notary Project as well as used by other open source projects and/or vendor tools that want to interoperate with Notary Project tooling.
 
-## Project Status
+Please see the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) file to learn about overall Notary Project.
 
-The Notary v2 project is in early development and design documents should not be considered final.
-Please refer to the [milestones](https://github.com/notaryproject/notaryproject/milestones) or [attend](#contributing--conversations) the weekly meetings for details on the roadmap.
+In this README
 
-## TOC
+- [Folder Structure](#folder-structure)
+- [Requirements](#requirements)
+- [Security Documents](#security-documents)
+- [Specifications](#specifications)
+- [Threat Models](#threat-models)
+- [Community](#community)
 
-- [Overview](#notary-v2-overview)
-- [Notary v2 Journey](#the-notary-v2-journey)
-- [Top Areas of Focus](#top-areas-of-focus)
-- [Stages of Development](#stages-of-development)
-- [Notary v2 February 2021 Status](./status-updates/notary-status-2021-02.md)
+## Folder Structure
 
-Additional details for Notary v2 efforts:
+| Folder Name   | Description  |
+| --------------| -------------|
+| [media](./media/)    | Media files referenced by documents in this repository |
+| [requirements](./requirements/) | Requirements for Notary Project |
+| [security](./security/) | Notary Project security related documents and reports |
+| [specs](./specs/) | Notary Project specifications |
+| [status-updates](./status-updates/) | This folder is not in active maintenance and contains status updates report for Notary Project |
+| [threatmodels](./threatmodels/) | Threat models for repositories under Notary Project |
 
-- [Goals & Requirements][nv2-requirements]
-- [Scenarios][nv2-scenarios]
-- [Threat Model][nv2-threat-model]
-- [Definitions & Terms][nv2-definitions]
-- [Contributing](#contributing-&-conversations)
+## Requirements
 
-## Notary v2 Overview
+| File Name     | Description |
+| -------- | ----------- |
+| [definitions-terms.md](./requirements/definitions-terms.md) | A collection of definitions and terms used within this repository |
+| [key-revocation.md](./requirements/key-revocation.md) | Requirements and proposals for key revocation |
+| [keymanagementrequirements.md](./requirements/keymanagementrequirements.md) | Requirements for key management |
```

Comment:
It's just a nit, but this file name has different pattern than other files. How about:
```suggestion
| [key-management-requirements.md](./requirements/key-management-requirements.md) | Requirements for key management |
```

Reply:
@Two-Hearts this requires changing the real file name, which is not the scope of this PR (update readme.md file). I would like to do this when addressing this issue https://github.com/notaryproject/.github/issues/48, and added a comment there for tracking https://github.com/notaryproject/.github/issues/48#issuecomment-1659591502

Reply:
I see. Then the comment can be resolved for now.